internals/secrethub/migrate_config_templates.go

```diff
@@ -89,7 +88,7 @@ func migrateTemplateTags(inFile io.Reader, outFile io.Writer, mapping referenceM
 		return nil, fmt.Errorf(errMsg)
 	}
 
-	_, err = io.WriteString(outFile, output)
+	err = ioutil.WriteFile(outFile, []byte(output), 0666)
```

Comment:
Ideally, it would preserve the file mode of the existing file.

Reply:
Adjusted to preserve file mode